javadoc.options

```diff
@@ -6,7 +6,7 @@
 -bottom 'Copyright &copy; Amazon.com, Inc. or its affiliates. All Rights Reserved.'
 -sourcepath src/main/java
 -notimestamp
--subpackages software.amazon.awssdk.crt
+-exclude software.amazon.awssdk.crt.internal
```

Comment:
did you diff the new and old docs to ensure we're not accidentally pulling in tests? or accidentally omitting major classes?

Reply:
good point, I ran the make doc again with java22, no major different in terms of classes. I did find some of the options are not applied. Updated those option to maven configuration instead.